docs/File-Formats.md

```diff
@@ -724,56 +724,93 @@ AKT1 AKT2 10000|AKT<TAB>0.17071492725<TAB>0.264067254391
+- **Site2_Entrez_Gene_Id** and/or **Site2_Hugo_Symbol**
+- **Site2_Exon**
+
+Description of all allowed columns:
+1. **Sample_ID**: Sample ID, as defined in the clinical sample file.
+2. **Site1_Entrez_Gene_Id**: An [Entrez Gene](http://www.ncbi.nlm.nih.gov/gene) identifier.
+3. **Site1_Hugo_Symbol**: A [HUGO](http://www.genenames.org/) gene symbol.
+4. **Site1_Ensembl_Transcript_Id**: An ENSEMBL Transcript ID.
+5. **Site1_Exon**: The exon of the canonical transcript.
+6. **Site1_Chromosome**: Chromosome.
+7. **Site1_Position**: Genomic position.
+8. **Site1_Description**: Decription of this event.
+9. **Site2_Entrez_Gene_Id**: An [Entrez Gene](http://www.ncbi.nlm.nih.gov/gene) identifier.
+10. **Site2_Hugo_Symbol**: A [HUGO](http://www.genenames.org/) gene symbol.
+11. **Site2_Ensembl_Transcript_Id**: An ENSEMBL Transcript ID.
+12. **Site2_Exon**: The exon of the canonical transcript.
+13. **Site2_Chromosome**: Chromosome.
+14. **Site2_Position**: Genomic position.
+15. **Site2_Description**: Description of this event
+16. **Site2_Effect_On_Frame**: `Frame_Shift` or `InFrame`.
+17. **DNA_Support**: `yes`/`no`.
+18. **RNA_Support**: `yes`/`no`.
+19. **Normal_Read_Count**: The total number of reads of the normal tissue.
+20. **Tumor_Read_Count**: The total number of reads of the tumor tissue.
+21. **Normal_Variant_Count**: The number of reads of the normal tissue that have the variant/allele.
+22. **Tumor_Variant_Count**: The number of reads of the tumor tissue that have the variant/allele.
+23. **Normal_Paired_End_Read_Count**: The number of paired-end reads of the normal tissue that support the call.
+24. **Tumor_Paired_End_Read_Count**: The number of paired-end reads of the tumor tissue that support the call.
+25. **Normal_Split_Read_Count**: The number of split reads of the normal tissue that support the call.
+26. **Tumor_Split_Read_Count**: The number of split reads of the tumor tissue that support the call.
+27. **Annotation**: Description of the gene or transcript rearrangement, i.e.:
+```
+PAX8 (NM_003466) rearrangement: c.1088-2028_c.25+11587del;
+TMPRSS2 (NM_001135099) - ERG (NM_182918) fusion: c.56-1943:TMPRSS2_c.18+13593:ERGdel
+```
+28. **Breakpoint_Type**: `PRECISE` or `IMPRECISE` which explain the resolution. Fill in `PRECISE` if the breakpoint resolution is known down to the exact base pair.
+29. **Center**: The sequencing center.
+30. **Connection_Type**: `3to5` or `5to3`. Which direction the connection is made
+31. **Event_Info**: Description of the event, such as `Fusion` or `Deletion of 7 exons`.
+32. **Class**: [`DELETION`, `DUPLICATION`, `INSERTION`, `INVERSION` or `TRANSLOCATION`]
+33. **Length**: Length of the structural variation in number ofbasepairs.
+34. **Comments**: Any comments or free text.
+35. **External_Annotation**: COSMIC or GENBank ID. Cosmic IDs follow the format `COSMIC:COS1000`.
+36. **cbp_driver**: "Putative_Passenger", "Putative_Driver", "Unknown", "NA" or "" (empty value).
```

Comment:
use the ` ` ` quotes

Reply:
also in lines below

Reply:
Actually, please remove this part of custom driver annotation. We will not be implementing any features for this in the next PRs.

Reply:
Done.